examples/app-autotune.yaml

```diff
@@ -5,7 +5,9 @@ metadata:
   namespace: "default"
 spec:
   slo:
-    objective_function: "transaction_response_time"
+    objective_function:
+      - type: expression
+      - expression: "transaction_response_time"
```

Comment:
array elements have to align right below the header
```
    objective_function:
    - type: expression
    - expression: "transaction_response_time"
```

Reply:
Updated